samples/ExportHistoryWebApp/ExportJobController.cs

```diff
@@ -0,0 +1,202 @@
+    /// <param name="createdFrom">Optional filter for jobs created after this time.</param>
+    /// <param name="createdTo">Optional filter for jobs created before this time.</param>
+    /// <param name="pageSize">Optional page size for pagination.</param>
+    /// <param name="continuationToken">Optional continuation token for pagination.</param>
+    /// <returns>A collection of export job descriptions.</returns>
+    [HttpGet("list")]
+    public async Task<ActionResult<IEnumerable<ExportJobDescription>>> ListExportJobs(
+        [FromQuery] ExportJobStatus? status = null,
+        [FromQuery] string? jobIdPrefix = null,
+        [FromQuery] DateTimeOffset? createdFrom = null,
+        [FromQuery] DateTimeOffset? createdTo = null,
+        [FromQuery] int? pageSize = null,
+        [FromQuery] string? continuationToken = null)
+    {
+        this.logger.LogInformation("GET list endpoint called with method: {Method}", this.HttpContext.Request.Method);
+        try
+        {
+            ExportJobQuery? query = null;
+            if (
+                status.HasValue ||
+                !string.IsNullOrEmpty(jobIdPrefix) ||
+                createdFrom.HasValue ||
+                createdTo.HasValue ||
+                pageSize.HasValue ||
+                !string.IsNullOrEmpty(continuationToken)
+            )
+            {
+                query = new ExportJobQuery
+                {
+                    Status = status,
+                    JobIdPrefix = jobIdPrefix,
+                    CreatedFrom = createdFrom,
+                    CreatedTo = createdTo,
+                    PageSize = pageSize,
+                    ContinuationToken = continuationToken,
+                };
+            }
+
+            AsyncPageable<ExportJobDescription> jobs = this.exportHistoryClient.ListJobsAsync(query);
+
+            // Collect all jobs from the async pageable
+            List<ExportJobDescription> jobList = new List<ExportJobDescription>();
+            await foreach (ExportJobDescription job in jobs)
+            {
+                jobList.Add(job);
+            }
+
+            return this.Ok(jobList);
+        }
+        catch (Exception ex)
+        {
+            this.logger.LogError(ex, "Error retrieving export jobs");
+            return this.StatusCode(500, "An error occurred while retrieving export jobs");
+        }
+    }
+
+    /// <summary>
+    /// Deletes an export job by its ID.
+    /// </summary>
+    /// <param name="id">The ID of the export job to delete.</param>
+    /// <returns>No content if successful.</returns>
+    [HttpDelete("{id}")]
+    public async Task<IActionResult> DeleteExportJob(string id)
+    {
+        this.logger.LogInformation("DELETE endpoint called for job ID: {JobId}", id);
+        try
+        {
+            ExportHistoryJobClient jobClient = this.exportHistoryClient.GetJobClient(id);
+            await jobClient.DeleteAsync();
```

Comment:
Do we need the pattern of getting a dedicated client for this? Or do you think we could wrap this into the `ExportHistoryClient`? Just wondering if we think it's worth it for the extra layer of abstraction?